scripts/core/editor3/helpers/insertAtomicBlockWithoutEmptyLines.ts

```diff
@@ -88,7 +88,7 @@ function insertAtomicBlockWithoutEmptyLines(editorState, entityKey, character) {
 
     var withAtomicBlock = Modifier.replaceWithFragment(asAtomicBlock, target.selectionState, fragment);
 
-    var newContent = withAtomicBlock.merge({
+    var newContent: ContentState|any = withAtomicBlock.merge({
```

Comment:
for some reason, the method `merge` returns a Map instead of a ContentState but it does act like a ContentState (see next lines, it is being used as a cs)

Reply:
so, the compiler didn't allow the map to be assigned to a content state and it didn't recognize the following methods used

Reply:
You can do `withAtomicBlock.merge() as ContentState` and the compiler won't complain.

Reply:
Merge returns a map because it's a method of immutablejs not draft-js, but I think it's compatible.